crates/daphne-service-utils/src/durable_requests/durable_request.capnp

```diff
@@ -35,10 +35,13 @@ struct DapAggregateShare @0xb34ce529a4a66aed {
     maxTime @2 :UInt64;
     checksum @3 :Data;
     data :union {
-        field64 @4 :Data;
-        field128 @5 :Data;
-        fieldPrio2 @6 :Data;
+        field64Draft09@4 :Data;
+        field128Draft09@5 :Data;
+        fieldPrio2Draft09@6 :Data;
         none @7 :Void;
+        field64 @8 :Data;
+        field128 @9 :Data;
+        fieldPrio2 @10 :Data;
```

Comment:
@mendess can you double check this renaming is safe? Not sure if the field can be renamed w/o side-effects to the serialization format.

Reply:
From the Cap'n Proto docs[1]
```
Any symbolic name can be changed, as long as the type ID / ordinal numbers stay the same. Note that type declarations have an implicit ID generated based on their name and parent’s ID, but you can use capnp compile -ocapnp myschema.capnp to find out what that number is, and then declare it explicitly after your rename.
```
[1] https://capnproto.org/language.html#evolving-your-protocol